src/BugYield/HarvestAdapter/HarvestAdapterApi.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace BugYield\HarvestAdapter;
+
+use Harvest\HarvestApi;
+use Harvest\Model\DayEntry;
+
+class HarvestAdapterApi extends HarvestApi {
+
+  /**
+   * Gets the entry specified
+   *
+   * Exists in adapter because: we want to have "multiple users"
+   * support (?of_user=).
```

Comment:
I actually had a fork ready with the first commit, but I've created two pull-requests now with these "changes". Thanks for reminding me :) :+1: